bundle-httpcache/src/main/java/com/adobe/acs/commons/httpcache/engine/impl/HttpCacheEngineImpl.java

```diff
@@ -30,23 +30,24 @@
  * to this.
  */
 // @formatter:off
-@Component(policy = ConfigurationPolicy.REQUIRE)
+@Component
 @Service
-@References({@Reference(name = HttpCacheEngineImpl.METHOD_NAME_TO_BIND_CONFIG,
-                        referenceInterface = HttpCacheConfig.class,
-                        policy = ReferencePolicy.DYNAMIC,
-                        cardinality = ReferenceCardinality.OPTIONAL_MULTIPLE),
-
-                        @Reference(name = HttpCacheEngineImpl.METHOD_NAME_TO_BIND_CACHE_HANDLING_RULES,
-                               referenceInterface = HttpCacheHandlingRule.class,
-                               policy = ReferencePolicy.DYNAMIC,
-                               cardinality = ReferenceCardinality.OPTIONAL_MULTIPLE),
-
-                        @Reference(name = HttpCacheEngineImpl.METHOD_NAME_TO_BIND_CACHE_STORE,
-                               referenceInterface = HttpCacheStore.class,
-                               policy = ReferencePolicy.DYNAMIC,
-                               cardinality = ReferenceCardinality.MANDATORY_MULTIPLE)})
-
+@References({
+        @Reference(name = HttpCacheEngineImpl.METHOD_NAME_TO_BIND_CONFIG,
```

Comment:
That boils to down to mandating atleast one cache config to be present to get this active. +1.